package.json

```diff
@@ -122,33 +120,29 @@
     "@types/qs": "^6.9.12",
     "@types/react": "^17.0.50",
     "@types/react-dom": "^17.0.17",
-    "@types/react-dropzone": "^4.2.2",
     "@types/react-gtm-module": "^2.0.3",
     "@types/react-helmet": "^5.0.8",
     "@types/react-router-dom": "^4.3.4",
-    "@types/react-sortable-hoc": "^0.7.1",
     "@types/url-join": "^4.0.1",
     "@types/uuid": "^9.0.4",
     "@types/webappsec-credential-management": "^0.5.1",
-    "@typescript-eslint/eslint-plugin": "^5.60.0",
-    "@typescript-eslint/parser": "^5.41.0",
+    "@typescript-eslint/eslint-plugin": "8.38.0",
+    "@typescript-eslint/parser": "8.38.0",
     "@vitejs/plugin-react-swc": "^3.8.0",
     "code-inspector-plugin": "^0.6.1",
     "core-js": "^3.7.0",
     "cross-env": "^6.0.3",
     "dependency-cruiser": "^12.10.0",
     "dotenv": "^10.0.0",
-    "eslint": "^8.43.0",
-    "eslint-config-prettier": "^8.5.0",
```

Comment:
I've checked our eslint config according to their docs: https://github.com/prettier/eslint-config-prettier#cli-helper-tool - and we don't have any rules that conflict with prettier, so it's safe to get rid of this.